modyn/protos/storage.proto

```diff
@@ -63,6 +63,9 @@ message GetDataPerWorkerRequest {
   string dataset_id = 1;
   int32 worker_id = 2;
   int32 total_workers = 3;
+  // value unset or set with default value means no limit
+  int64 start_timestamp = 4;
```

Comment:
In `proto3` there is no "required" field. A field without `optional` prefix means if the field is unset, the default value is returned every time we read this `field`.

I don't have a specific reason: I just find it very handy to automatically have the default value when unspecified, rather than having to check if the field is present on the reader side.

More info https://protobuf.dev/programming-guides/field_presence/#c-example